tensorflow/python/data/experimental/ops/readers.py

```diff
@@ -340,7 +340,28 @@ def make_csv_dataset_v2(
   tuple that corresponds to a batch of CSV rows. The features dictionary
   maps feature column names to `Tensor`s containing the corresponding
   feature data, and labels is a `Tensor` containing the batch's label data.
-
+  
+  Usage Example:
+  
+  Using IRIS dataset to show how to convert .csv file into a dataset.
+  
+  ```python
```

Comment:
I'd recommend converting this to a doctest, see guidelines at https://www.tensorflow.org/community/contribute/docs_ref#testable_docstrings

Reply:
Ah, I wasn't aware that `>>>` == doctest earlier, and requested for the author to use backtick notation for consistency with the rest of the file. Apologies for the confusion.